apps/src/sites/studio/pages/initApp.js

```diff
@@ -70,7 +70,7 @@ window.apps = {
         $(document).trigger('appInitialized');
       },
       onAttempt: function (report) {
-        if (appOptions.level.isProjectLevel) {
+        if (appOptions.level.isProjectLevel && !appOptions.level.edit_blocks) {
```

Comment:
For project levels, the attempt isn't reported until you hit the finish button. We don't show the finish button when editing project level templates (e.g. "New Play Lab Project"), so it needs to report the attempt here to be able to save changes.